pkgs/applications/editors/mle/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, termbox, pcre, uthash, lua5_3 }:
+
+stdenv.mkDerivation rec {
+  pname = "mle";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "adsr";
+    repo = "mle";
+    rev = "v${version}";
+    sha256 = "1yz6yrnib3k7kg30v2sllc13y328faf8k3nyh1y2s3rinf5fyzs2";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile --replace "-llua5.3" "-llua";
```

Comment:
Might be worth putting some motivation for these somewhere, did you create this fix yourself or is it from somewhere else

Reply:
Have just realised this is your own project, is there any reason this can't be upstreamed?

Reply:
The version in trunk (`-llua5.3` / `<lua5.3/lua.h>`) works with Debian-based systems. Alternatives to this are using `pkg-config` or trying to guess the location of Lua dynamically. Personally I prefer this small patch as I find it the simplest to read.